src/coffee/directives/api/map.coffee

```diff
@@ -101,7 +101,10 @@ angular.module('uiGmapgoogle-maps.directives.api')
               zoom: scope.zoom
               bounds: scope.bounds
 
-            _gMap = new google.maps.Map(el.find('div')[1], mapOptions)
+            if typeof geeCreateFusionMap is 'function'
```

Comment:
This is very specific to your needs and not for the general public. If anything there should be a MapFactory which can call the default `new google.maps.Map`. Then you could use the decorator pattern in Angular to override the MapFactory to your custom behavior below.

http://blog.xebia.com/2014/08/08/extending-angularjs-services-with-the-decorate-method/